README.md

```diff
@@ -1,22 +1,49 @@
 # Robot_Dog
-Four legged robot URDF in Rviz and Gazebo [Video](https://youtu.be/crmAcT25vDQ) <br/>
+Four legged robot URDF in Rviz and Gazebo [Video](https://youtu.be/SznYLX9XDVU) <br/>
 ![](https://github.com/mohamedsayed18/Robot_Dog/blob/Readme/images/Dog_Gazebo.png)
 ### How to run
 * Clone the repo `git clone https://github.com/mohamedsayed18/Robot_Dog.git`<br/>
 * Put the packages (spot, rrbot_control) in your work space <br/>
 * build the work space `catkin_make`<br/>
 * then write this command <br/>
+Run all from one launch <br/>
+`roslaunch spot one_for_all.launch`<br/>
+
+Or run them separately <br/>
 launch the robot in gazebo<br/>
 `roslaunch spot dog_world.launch`<br/>
 launch the control files<br/>
 `roslaunch rrbot_control rrbot_control.launch`<br/>
 Run this script which send values to the joints<br/>
 `rosrun rrbot_control publisher.py`<br/>
+Run the Subscriber
+`rosrun spot camera_sub.py`<br/>
+
+Run tests
+
+unit test `rosrun tests test_talker.py`<br/>
+integration test `rostest tests integration.launch`
 
 ### Control in Gazebo
 i followed this [tutorial](http://gazebosim.org/tutorials/?tut=ros_control)<br/>
 this [repo](https://github.com/ros-simulation/gazebo_ros_demos.git) contain examples which they use in the above tutorial<br/>
 
+### adding camera
+i followed this [Tutorial: Using Gazebo plugins with ROS](http://gazebosim.org/tutorials?tut=ros_gzplugins)<br/>
```

Comment:
Please remove all build stuff. These files not needed to be in repo. And why camera PR is here too? Fix please.

Reply:
edited, but i don't know why some commits appear in the both PR (testes, camera)

Reply:
:( and now 557 files are changed. And after merge you will not see files because you removed them. You see both changes because you created your branch based on camera branch, but should be based on master.

Reply:
i don't notice that, i will try to make it work